doc/developers.md

```diff
@@ -50,24 +50,16 @@ replay the new events with no more interactions with OpenMRS.
 
 ## Running end-to-end tests
 
-Parquet tools library is used to inspect parquet files. The jar file has been
-included in the project under `/e2e-tests/parquet-tools-1.11.1.jar`
+When a Pull Request is submitted, it triggers a request to Google-owned project running [Cloud Build](https://cloud.google.com/build) to run the build config file defined in [`cloudbuild.yaml`](../cloudbuild.yaml). The CI pipeline can also be run locally, using the Cloud Build local builder. To setup the local builder, follow the instructions [here](https://cloud.google.com/build/docs/build-debug-locally); you will need Docker, and the Google Cloud SDK to install the local builder. Once the local builder is installed, do the following:
 
-To regenerate this jar file:
+1. Stop any running `openmrs`, `openmrs-fhir-mysql`, and `sink-server` containers by running:
 
-1.  Clone [parquet-mr](https://github.com/apache/parquet-mr)
-2.  Checkout last released version `git checkout apache-parquet-1.11.1`
-3.  [Install](https://github.com/apache/parquet-mr#install-thrift) thrift
-    compiler v0.12.0
-4.  To build the jar file run `mvn -pl parquet-tools -am clean install -Plocal
-    -DskipTests`
-    You should be able to see `parquet-tools-1.11.1.jar` inside parquet-tools
-    module.
-5.  Command usage for parquet tools `java -jar ./parquet-tools-<VERSION>.jar
-    <command> my_parquet_file`
-
-    NOTE: Parquet tools will be replaced with parquet cli in the next release
-    `apache-parquet-1.12.0`
+    ```bash
+    docker stop sink-server openmrs openmrs-fhir-mysql
+    ```
 
-To run the e2e tests, execute ```sh /e2e-tests/run.sh```
+2. Run the e2e-test using the local builder:
 
+    ```bash
+    cloud-build-local  --dryrun=false .
```

Comment:
Can devs use your new `run-batch-local.sh` to speed up this process; if yes, please document it.

Reply:
No, the use case for run_batch_local.sh script is during development. If a developer has the server up, and the code compiled, there's no need for them to run the e2e-test via the cloud-build-tool. Instead, they can run run_batch_local.sh` to build and run the batch pipeline

Reply:
`cloud-build-local` should ideally only be used for e2e-testing only, and should be re-run every-time you make changes. 

`run_batch_local.sh` assumes you have the OpenMRS server up and running. It is basically a wrapper for running:

```
java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
    org.openmrs.analytics.FhirEtl \
    --openmrsServerUrl=http://localhost:8099/openmrs \
    --openmrsUserName=admin --openmrsPassword=Admin123 \
    --fhirSinkPath=http://localhost:8098/fhir \
    --sinkUserName=hapi --sinkPassword=hapi \
    --outputParquetPath=/tmp/TEST/ \
    --resourceList=Patient,Encounter,Observation --batchSize=20
```

and want to see what happens.